src/storybook/styles/table/readme.md

```diff
@@ -0,0 +1,72 @@
+The `sbb-table` class is designed to handle the styling of common table use cases, ensuring a consistent and visually appealing presentation of tabular data.
+This guide will walk you through the various customization options and advanced scenarios for styling tables within our design system.
+
+### Basic usage
+
+```html
+<table class="sbb-table">
+  <thead>
+    <th>...</th>
+    ...
+  </thead>
+  <tbody>
+    <tr>
+      <td>...</td>
+      ...
+    </tr>
+    ...
+  </tbody>
+  <caption>
+    ...
+  </caption>
+</table>
+```
+
+### Striped table
+
+By default, tables are styled with alternating row stripes to enhance readability. To remove this striping, add the `sbb-table-unstriped` class:
+
+```html
+<table class="sbb-table sbb-table-unstriped">
```

Comment:
I'm not sure what is best, I only remember for the sbb-text--bold, we used '--' to show that it can't be used standalone and is only a modifier of the class. (this comment would also apply to the sizes and negative) Probably to be discussed.

```suggestion
<table class="sbb-table sbb-table--unstriped">
```

Reply:
I think I agree with you. I only see an inconsistency in the size classes since the text ones are like `sbb-text-${size}`.
I'll change negative and stripe for now, we can discuss the size eventually

Reply:
yeah but the sbb-text sizes do include all the rules, so it could be used standalone. We could also have this approach in table, but would generate more output

Reply:
Mmm ok. So we do not have an inconsistency if we change the table size classes to `sbb-table--*size*`.
I'd rather avoid making them standalone because I expect that the table CSS will grow in size with the advanced features

Reply:
The specification intended for sbb-table, sbb-table-s and sbb-table-m to be usable standalone.
I.e. you could use `sbb-table` or `sbb-table-m`.
But I'm open for discussion.

Reply:
I've misinterpreted the specification :/
In terms of output, it doesn't change much to have them standalone (if we implement them like this)

Note: Bootstrap combines them 

![image](https://github.com/sbb-design-systems/lyne-components/assets/12052575/747cf385-c4bb-45c4-a6da-8c6003f6c407)